### PR TITLE
chore: add missing "T"

### LINKED ISSUE #11848: Add missing "T" to issue template (maintenance)
## Description

"here are no" misses the "T"

https://github.com/foundation/foundation-sites/blob/develop/.github/ISSUE_TEMPLATE/4-maintainance.md

## Possible Solution

...

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] here are no other issues similar to this one.
- [ ] The issue title and template are correctly filled.

### PR DESCRIPTION
**Pull request by jfamousket**
## Description

adding a missing 'T' in 4_maintainance.md

- Closes https://github.com/foundation/foundation-sites/issues/11848

## Types of changes

- [x] Documentation
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (anything that would change an existing functionality)
- [ ] Maintenance (refactor, code cleaning, development tools...)

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] The pull request title and template are correctly filled.
- [x] The pull request targets the right branch (`develop` or `develop-v...`).
- [x] My commits are correctly titled and contain all relevant information.
- [ ] I have updated the documentation accordingly to my changes (if relevant).
- [ ] I have added tests to cover my changes (if relevant).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
 ---
 name: "\U0001F4BB Maintainance"
 about: Tests, dependencies, workflow...
-
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: Please do not remove this line, the other templates have this line too =)

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 2]
```diff
@@ -16,25 +15,25 @@ about: Tests, dependencies, workflow...
 
 ...
 
-
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: We should keep this line too.

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 3]
```diff
@@ -16,25 +15,25 @@ about: Tests, dependencies, workflow...
...
 ...
 
-
 ## Checklist
+
 <!-------------------------------------------------------------------
 │   Please ensure that all the following points are respected.
 │   Fill with [x] the boxes once the rule is respected.
 └------------------------------------------------------------------->
+
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: I don't think this is really needed.

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 4]
```diff
@@ -16,25 +15,25 @@ about: Tests, dependencies, workflow...
...
 <!-------------------------------------------------------------------
 │   What do you suggest to resolve this issue?
 └------------------------------------------------------------------->
 
 ...
 
-
 ## Checklist
+
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: I don't think this is really needed.

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 5]
```diff
@@ -16,25 +15,25 @@ about: Tests, dependencies, workflow...
 
 ...
 
-
 ## Possible Solution
+
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: I don't think this is really needed.

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 6]
```diff
@@ -16,25 +15,25 @@ about: Tests, dependencies, workflow...
...
 │   Please ensure that all the following points are respected.
 │   Fill with [x] the boxes once the rule is respected.
 └------------------------------------------------------------------->
+
 - [ ] I have read and follow the CONTRIBUTING.md document.
-- [ ] here are no other issues similar to this one.
+- [ ] There are no other issues similar to this one.
 - [ ] The issue title and template are correctly filled.
 
-
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: Please keep that line for now, thanks.

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 7]
```diff
@@ -16,25 +15,25 @@ about: Tests, dependencies, workflow...
...
 ## Checklist
+
 <!-------------------------------------------------------------------
 │   Please ensure that all the following points are respected.
 │   Fill with [x] the boxes once the rule is respected.
 └------------------------------------------------------------------->
+
 - [ ] I have read and follow the CONTRIBUTING.md document.
-- [ ] here are no other issues similar to this one.
+- [ ] There are no other issues similar to this one.
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: :+1:

**Review by DanielRuf — CHANGES_REQUESTED** [review]

**Comment by DanielRuf** [comment]
Hi @jfamousket,

thanks for adding the missing letter. Can you add ` - closes #11848` to the commit message using `git commit --amend && git push --force-with-lease`? That would be awesome =)

**Comment by jfamousket** [comment]
> Hi @jfamousket,
> 
> thanks for adding the missing letter. Can you add ` - closes #11848` to the commit message using `git commit --amend && git push --force-with-lease`? That would be awesome =)

Done :)

File: .github/ISSUE_TEMPLATE/4-maintainance.md (hunk 5 above)
Comment (jfamousket) [reply .github/ISSUE_TEMPLATE/4-maintainance.md]: sorry Prettier changes

**Comment by DanielRuf** [comment]
Thanks, I will squash the commits so we have only one commit.

**Review by DanielRuf — APPROVED** [review]
👏

**Pull request merged**